iosApp/iosAppTests/Views/ContentViewTests.swift

```diff
@@ -56,8 +57,9 @@ final class ContentViewTests: XCTestCase {
 
         ViewHosting.host(view: sut)
 
-        try sut.inspect().implicitAnyView().view(ContentView.self).implicitAnyView().implicitAnyView().implicitAnyView()
-            .vStack().callOnChange(newValue: ScenePhase.background)
+        try sut.inspect().implicitAnyView().view(ContentView.self).implicitAnyView()
+            .vStack()
+            .callOnChange(newValue: ScenePhase.background)
         wait(for: [disconnectedExpectation], timeout: 1)
         try sut.inspect().implicitAnyView().view(ContentView.self).implicitAnyView().implicitAnyView().implicitAnyView()
             .vStack().callOnChange(newValue: ScenePhase.active)
```

Comment:
The second `callOnChange` in `testReconnectsSocketAfterBackgroundingAndReactivating` needs to also be trimmed down to one `implicitAnyView` for this test to pass in Xcode 16.

Reply:
👍 if you have that change locally, could you put up a separate PR for it and I'll ✅? That way we can get this merged & start the CI for a new build ASAP

Reply:
(or I can put it up if you don't have it locally)